react/src/components/PortalAccessKey.js

```diff
@@ -47,9 +47,13 @@ const PortalAccessKey = (props) => {
             <tr>
                 <td style={tdlabel}>Creation Date:</td>
                 <td style={tdstyle}>
-                    {accessKey.created_at}
-                    &nbsp;{Char.RightArrow}
-                    &nbsp;<small>{Time.Ago(accessKey.created_at, true, false)}</small>
+                    { accessKey.created_at ? <>
+                        {accessKey.created_at}
+                        &nbsp;{Char.RightArrow}
```

Comment:
This is what was there before, and maybe it's obvious when you see it, but this notation seems questionable. Somehow I expect "ago" to appear at the end of the text to make it make sense and to avoid ambiguity with "left" (or "remaining"). Does this function `Time.Ago` append that word? Why is the time since creation and not the time left the relevant thing to show?

Reply:
The Time.Ago function takes a datetime and returns a string which is the formatted time duration from that date until "now" and with the word "ago" append, e.g. the whole line looks like `Creation Date: | 2023-01-19 16:35:50 → 145 days 18 hours 34 minutes ago`